### PR TITLE
[processor/deltatocumulative]: alpha stability

### PR DESCRIPTION
**Pull request by tombrk**
**Description:** most major functionality is implemented and ready for wider testing. lets release as alpha and gather feedback

**Link to tracking Issue:** https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/30705

### CODE REVIEW

**Comment by sh0rez** [comment]
@jpkrohling does this need a changelog?

**Comment by jpkrohling** [comment]
Yes, we need to communicate to users that this is now available as alpha.

File: .chloggen/deltatocumulative-alpha.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: deltatocumulative
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: alpha stability
```
Comment (ArthurSens) [inline .chloggen/deltatocumulative-alpha.yaml:10]:
```suggestion
note: Promote to `alpha` stability
```

this is 100% nitpicking :P I was just looking how stability changes are announced in the changelog

File: .chloggen/deltatocumulative-alpha.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [34747]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: promotes the deltatocumulative processor to alpha stability, as most features are now implemented and ready for wider testing
```
Comment (ArthurSens) [inline .chloggen/deltatocumulative-alpha.yaml:18]:
```suggestion
subtext: promotes the deltatocumulative processor to alpha stability, as most features are now implemented and ready for wider testing.
```

**Review by ArthurSens — APPROVED** [review]
some small nitpicks 😬

**Comment by crobert-1** [comment]
From failing CI action:
```
Generated code is out of date, please run "make gendistributions" and commit the changes in this PR.
```

**Review by RichieSams — APPROVED** [review]

**Review by crobert-1 — APPROVED** [review]
LGTM.

Just in case it isn't clear, to be able to have this component included in releases of the `contrib` distribution you'll also need to open a PR against the [releases repository](https://github.com/open-telemetry/opentelemetry-collector-releases) as described [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#releasing-new-components). 👍

**Review by jpkrohling — APPROVED** [review]

**Comment by sh0rez** [comment]
PR against the releases repo opened: https://github.com/open-telemetry/opentelemetry-collector-releases/pull/647

**Pull request merged**